### ISSUE TITLE
Sometimes pages Not Found will occur if click "中文" Button

### ISSUE THREAD

**Issue by drelaptop** [cocos2d/cocos2d-x-docs#71]
When user reads English docs the `getting_started` page of every chapter.

such as, click  "中文" Button on [http://cocos2d-x.org/docs/en/basic_concepts/getting_started.html](http://cocos2d-x.org/docs/en/basic_concepts/getting_started.html)

**Comment by drelaptop** [comment]
Chinese docs didn't have `getting_started.html`, It need to been solved but not urgent. 

Because it's a seldom occur for English Reader click "中文" button, and `getting_started.html` content is little.

**Comment by slackmoehrle** [comment]
Agreed, not a high priority. I can always do this for you if you want. Of you could just copy EN docs there and translate later if you get more concerned.

**Comment by drelaptop** [comment]
Thanks, but I can do it by myself, just do your business I know you are busy than me.

**Comment by slackmoehrle** [comment]
This is fixed now, correct?

**Comment by drelaptop** [comment]
No, I don't fixed it for now, I think it's not necessary, except developer thinks it will case doubt.

**Comment by slackmoehrle** [comment]
This is still technically an issue, no 404's now, just an incorrect page. I can work on re-organizing a bit as time allows.
